### ISSUE TITLE
[Accessibility]: 'Weather card' is not properly visible after resizing the application under 'Weather' pane.

### ISSUE THREAD

**Issue by RebeccaAnne** [microsoft/AdaptiveCards#6262]
### Target Platforms

Android

### SDK Version

Application Version: 2021.06.04.1

### Application Name

Visualizer

### Problem Description

[34087027](https://microsoft.visualstudio.com/OS/_workitems/edit/34087027)

'Weather card' should be properly visible after resizing the application under 'Import' tab as it is visible in normal mode.

### Card JSON

[WeatherLarge.json](https://github.com/microsoft/AdaptiveCards/blob/main/samples/v1.0/Scenarios/WeatherLarge.json)
